plugins/modules/system/parted.py

```diff
@@ -557,12 +557,12 @@ def parted(script, device, align):
         align_option = ''
 
     if script and not module.check_mode:
-        command = "%s -s -m %s %s -- %s" % (parted_exec, align_option, device, script)
+        command = [parted_exec, '-s', '-m', align_option, device, '--', script]
```

Comment:
Shouldn't it be
```suggestion
        command = [parted_exec, '-s', '-m', align_option, device, '--'] + script
```
as `script` is a list?

I think someone needs to verify whether the module still works after this refactor, just to make sure there aren't more surprises :)